pkg/controller/controller_serviceplan_test.go

```diff
@@ -18,6 +18,7 @@ package controller
 
 import (
 	"errors"
+	"k8s.io/client-go/tools/cache"
```

Comment:
**nit:** it will be good to have imports sorted

Reply:
imports fixed